deploy/charts/harvester-crd/README.md

```diff
@@ -0,0 +1,58 @@
+# Harvester CRD Helm Chart
+
+## Chart Details
+
+This chart includes following CRDs:
+
+- Harvester (Automatically generated by [this script](https://github.com/harvester/harvester/blob/6557fdfd178b149b2af48ca0c8274bd3234c2066/scripts/generate-manifest))
```

Comment:
Could you change this link to master branch?

Reply:
For other links, I can't use master, so I leave it as is.